nastaveni/nastaveni.php

```diff
@@ -239,5 +239,6 @@
 define('SUPERADMINI', [1682, 4032]);
 
 @define('ADRESAR_WEBU_S_OBRAZKY', __DIR__ . '/../web');
+@define('ZALOHA_DB_SLOZKA', __DIR__ . '../backup/db'); // cesta pro zálohy databáze
```

Comment:
Nechybí ti tam náhodou lomítko (všude jinde je a jestli se nepletu, tak nemáš rád koncová lomítka, takže __DIR__ by na lomítko končit nemělo) 😄 
```suggestion
@define('ZALOHA_DB_SLOZKA', __DIR__ . '/../backup/db'); // cesta pro zálohy databáze
```

Reply:
Přesně tak:D